src/main/java/de/metas/ui/web/picking/husToPick/process/WEBUI_HUsToPick_PickCU.java

```diff
@@ -221,6 +245,19 @@ private void pickCUs()
 
 		final I_M_HU splitCU = ListUtils.singleElement(splitHUs);
 		addHUIdToCurrentPickingSlot(splitCU.getM_HU_ID());
+
+		// #3778
+		// Process the picking automatically in case it is configured this way.
+
+		isAutoProcess = pickingConfigRepo.getPickingConfig().isAutoProcess();
```

Comment:
methods shall not have side effects, uncle bob says.
Hence, getQtyCU() shall not set the `isAutoProcess` field. Better would be to set the isAutoProcess flag as first line in doIt().